common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -112,7 +61,7 @@ public ClassType getClassType() {
         return classType;
     }
 
-    public boolean isReskinned() {
+    private boolean isReskinned() {
```

Comment:
Are there conceptual reasons for making this private, or is it just that currently there are no external users but it is really part of the API we expose? If it is the latter, I think it can remain public.

Reply:
The latter, I'll change it back